site/layouts/partials/footer.html

```diff
@@ -44,5 +44,58 @@
     </script>
     {{ end }}
 
+    <script>
+    document.addEventListener('DOMContentLoaded', async () => {
+        const elems = document.querySelectorAll(".krew-plugin-count");
+        if (elems.length === 0){
+            return;
+        }
+        let count = '<error>';
+        try {
+            const resp = await fetch('/.netlify/functions/api/pluginCount');
+            const json = await resp.json();
+            count = json.data?.count ?? '<error>';
+        } catch (e) {
+            console.warn(e);
+        }
+        elems.forEach(elem => elem.innerText=count);
+    });
+    </script>
+
+    <script type="module">
+    import {html, render} from 'https://unpkg.com/lit-html?module';
```

Comment:
I must admit I don't fully understand how imports work in the browser so @corneliusweig you’ll probably be on point for maintaining these :D I barely can get the vanilla js working.

Reply:
Yeah, I'm not too worried about it ((support stats)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import]).